applications/asset_tracker/src/main.c

```diff
@@ -948,6 +948,36 @@ static void modem_configure(void)
 #endif
 }
 
+/**@brief Initializes the accelerometer device and
+ * configures trigger if set.
+ */
+static void accelerometer_init(void)
+{
+	struct device *accel_dev =
+	device_get_binding(CONFIG_ACCEL_DEV_NAME);
+
+	if (accel_dev == NULL) {
+		printk("Could not get %s device\n", CONFIG_ACCEL_DEV_NAME);
+		return;
+	}
+
+	struct sensor_trigger sensor_trig = {
+		.type = SENSOR_TRIG_THRESHOLD,
+	};
+
+	if (!IS_ENABLED(CONFIG_FLIP_POLL) && IS_ENABLED(ACCEL_USE_EXTERNAL)) {
+		printk("Setting trigger \n");
+		int err = 0;
+
+		err = sensor_trigger_set(accel_dev, &sensor_trig,
+				sensor_trigger_handler);
+
+		if (err) {
+			return;
```

Comment:
We are returning after this function call anyway.

Also, can we do better than silently ignoring the error?

Reply:
Fixed.